Code/GraphMol/MolDraw2D/Wrap/testMolDraw2D.py

```diff
@@ -622,5 +622,41 @@ def testUpdateParamsFromJSON(self):
     txt = d2d.GetDrawingText()
     self.assertTrue('>8</text>' in txt)
 
+  def testIsotopeLabels(self):
```

Comment:
This test seems way too indirect and complicated. 
Can't you just create the `MolDraw2DSVG` objects with the `noFreetype` ctor argument (the last one) set to `True` and then look in the output SVG for the strings which should/should not be there?

Reply:
I knew you'd complain after I finished to write those tests :). OK, I will make them simpler and more direct.

Reply:
yeah, I'm predictable. :-)

Reply:
Done - I am now counting matches to a simple regex.